# Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.

# Key-value entries in the proto2ros/Struct ROS 2 message. Equivalent to
# the auxiliary Protobuf message used over-the-wire representation of the
# google.protobuf.Struct message.

string key
proto2ros/Value value
